core/domain/collection_services.py

```diff
@@ -702,21 +877,41 @@ def update_collection(
 
 
 def create_collection_summary(collection_id, contributor_id_to_add):
-    """Create summary of a collection and store in datastore."""
+    """Create summary of a collection and store in datastore.
```

Comment:
Let's standardize the tense for these docstrings: Creates and stores a summary of the given collection.

Ditto elsewhere.